inc/html.php

```diff
@@ -74,7 +74,7 @@ function html_login(){
 function html_denied() {
     print p_locale_xhtml('denied');
 
-    if(!$_SERVER['REMOTE_USER']){
+    if(!isset($_SERVER['REMOTE_USER'])){
```

Comment:
wouldn't empty($_SERVER['REMOTE_USER'] be a better replacement?

Can $_SERVER['REMOTE_USER'] be set and empty?